conda-forge.yml

```
@@ -1,7 +1,13 @@
-build_platform: {osx_arm64: osx_64, linux_ppc64le: linux_64}
+build_platform:
+  linux_aarch64: linux_aarch64
+  linux_ppc64le: linux_ppc64le
+  osx_arm64: osx_64
 conda_forge_output_validation: true
-provider: {linux_aarch64: default, linux_ppc64le: azure, win: azure}
-test_on_native_only: true
 github:
   branch_name: main
   tooling_branch_name: main
+provider:
+  linux_aarch64: default
+  linux_ppc64le: azure
```

```suggestion
  linux_aarch64: default
  linux_ppc64le: default
```

and rerender. This should make them both compile through travis. Hopefully qemu won't be a problem there. You have a 1 hour time limit. So maybe you'll get in right under it?